brave/brave-core
# Splited p3a from crash report option from first run dialog

p3a prefs would not be affected by first run dialog.
crash report checkbox is unchecked by default.

fix https://github.com/brave/brave-browser/issues/14183
fix https://github.com/brave/brave-browser/issues/14160

![image](https://user-images.githubusercontent.com/6786187/108145278-846aba80-710e-11eb-9e99-7972e35253e0.png)
![image](https://user-images.githubusercontent.com/6786187/108145503-e0354380-710e-11eb-805d-a40cf058033d.png)


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Launch with clean profile
2. Check crash report checkbox is un-checked by default
3. Check settings reflects properly for crash report checkbox state
3. Check crash report checkbox state doesn't affect p3a prefs state
